packages/server/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "../../tsconfig.json",
   "include": ["src", "test"],
   "compilerOptions": {
-    "lib": ["ES2020", "DOM"],
+    "lib": ["ES2020", "DOM", "DOM.Iterable"],
```

Comment:
Do we need this when we are targeting node 14? 🤔 

Related: https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping#node-14

Reply:
Yes, it is needed to add `Symbol.iterator` to `Headers` so we can use `Object.fromEntries`:
https://github.com/trpc/trpc/blob/dcc48da36ca3056b2747fb95303ed9edf1c97ce5/packages/server/src/adapters/fetch/fetchRequestHandler.ts#L27

Note that Node does not implement the Fetch spec.